pycsob/utils.py

```diff
@@ -165,3 +167,29 @@ def convert_keys_to_camel_case(data: T) -> T:
         else:
             converted_dict[key] = value
     return converted_dict
+
+
+def get_customer_data_signature_message(customer_data: dict[str, Any]) -> str:
+    """
+    Returns signature string from customer data used to sign the request.
+    For more information follow the API documentation
+    https://github.com/csob/platebnibrana/wiki/Podpis-po%C5%BEadavku-a-ov%C4%9B%C5%99en%C3%AD-podpisu-odpov%C4%9Bdi
+    """
+
+    def get_joined_values(data: dict[str, Any], keys: list) -> str:
+        """
+        Args:
+            data: payload customer data
+            keys: list with ordered keys
+        """
+        return "|".join(str_or_jsbool(data[key]) for key in keys if key in data)
+
+    customer_keys = ["name", "email", "mobilePhone"]
```

Comment:
Nitpick: I can see you are mixing single quote and double quote conventions. I know it is pia without automation so I let it up to you, just pointing this out.

Reply:
You are right, i've changed it to single quotes 👍